### PR TITLE
Add Iosevka w/ git auto-update

### PR DESCRIPTION
**Pull request by aymanbagabas**
Use the mirror https://github.com/aymanbagabas/iosevka-fonts instead of https://github.com/be5invis/Iosevka to distribute the fonts.

### CODE REVIEW

File: packages/i/Iosevka.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "license": "OFL-1.1",
+  "keywords": ["font"],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/aymanbagabas/iosevka-fonts.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": ["*/*.css", "*/*/.(ttf|woff2)"]
```
Comment (alexeyten) [inline packages/i/Iosevka.json:18]: Shouldn't it be `*/*/*.(ttf|woff2)`?

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
I am very hesitant to accept a PR where the target for auto-updating is known to not be the true source -- there is no history to demonstrate that this source can be trusted, you could easily publish files that are not in the true source.

**Comment by aymanbagabas** [comment]
> I am very hesitant to accept a PR where the target for auto-updating is known to not be the true source -- there is no history to demonstrate that this source can be trusted, you could easily publish files that are not in the true source.

I totally agree. This is related to https://github.com/be5invis/Iosevka/issues/942. The reason for this is that the author refused to maintain the files in the main repository.

**Review by MattIPv4 — APPROVED** [review]
Lgtm, seems the original maintainer is okay with this.

**Pull request merged**